Deferred/LockProtected.swift

```diff
@@ -6,8 +6,6 @@
 //  Copyright © 2014-2015 Big Nerd Ranch. Licensed under MIT.
 //
 
-import Foundation
-
```

Comment:
Discussed in the comments on #13 (the Swift 1.x version of this PR). We should discuss in Slack in the morning.

/cc @jeremy-w